Unity-Technologies/Graphics
# [XPipeline]Cleaning Light Inspector Code and using SerializedLight

### Purpose of this PR

Prepare URP Light Inspector for the Unification with HDRP

Creating the Interface `ISerializedCamera` and implement it on both pipelines

On URP using the `UniversalRenderPipelineSerializedCamera`

[JIRA](https://jira.unity3d.com/browse/XPIPELINE-144)

---
### Testing status
- Check the animations are deleted
- Check that the values are changed properly(Multiple inspectors opened)

---
### Comments to reviewers
- UX do not want animations when show/hide elements, I've removed them
- First step towards the unification of the Light Inspectors
- The Additional data is always initialized, no need to check if it null on the shadow properties, this simplifies a lot the code
